andela/travel_tool_back
# #162636597 Create seeders for e2e tests

#### What does this PR do?
Create seeders to be used for end 2 end tests

#### Description of Task to be completed?
- Create user seeder
- Create role seeder
- Create user-role seeder
- Create occupations seeder
- Create checklist item seeder

#### How should this be manually tested?
- Clone the Travela backend repo ```git clone https://github.com/andela/travel_tool_back.git```
- Open the project in your text editor
- Checkout to this branch ```git checkout ch-create-e2e-seeders-162636597```
- Run the command below on your terminal to rollback, migrate and seed e2e data to your database
```yarn db:rollback && yarn db:migrate && yarn db:seed:e2e```
- Check your database to confirm that the user table, roles table, requests table, and user-roles tables have been populated with data.


#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#162636597](https://www.pivotaltracker.com/n/projects/2184887/stories/162636597)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
